pkg/generate/app/environmentfile.go

```diff
@@ -0,0 +1,37 @@
+package app
+
+import (
+	"bufio"
+	"os"
+)
+
+// IsPossibleEnvironmentFile returns true if the argument is a file. I am not
+// entirely sure, why we are using this function, as this is just a wrapper,
+// we are not doing any additional validations here. This is something that I
+// am borrowing from templates file. Need feedback on this.
+func IsPossibleEnvironmentFile(value string) bool {
+	var fileexists bool = true
+	_, err := os.Stat(value)
+	if err != nil {
+		fileexists = false
+	}
+	return isFile(value) && fileexists
+}
+
+// GetEnvironmentVars reads a whole file into memory and returns string
+// array. This could be a problem if we are reading a very large file. I am
+// not putting any validation for file, if it contains a key=value pair as
+// that should be handled in environment variables that we are passing.
+func GetEnvironmentVars(path string) []string {
+	envFile, err := os.Open(path)
+	if err != nil {
+		return nil
```

Comment:
Agree with you, the reason I had to do this was because of having an additional check earlier(IsEnvironmentFile) or else I had to open the file twice. The other thing that I had to create IsEnvironmentFile is because of having consistency - templatefile is doing something similar. But I will make those changes and update the PR.